README.md

```diff
@@ -246,13 +246,27 @@ Source.fromPublisher(consumerWithOffsetSink.publisher)
 Tuning
 ----
 
-KafkaActorSubscriber and KafkaActorPublisher have their own thread pools, configured in `reference.conf`.
+`KafkaActorSubscriber` and `KafkaActorPublisher` have their own thread pools, configured in `reference.conf`.
 You can tune them by overriding `kafka-publisher-dispatcher.thread-pool-executor` and
 `kafka-subscriber-dispatcher.thread-pool-executor` in your `application.conf` file.  
-Alternatively you can provide your own dispatcher name. It can be passed to appropriate variants of factory methods in
+Alternatively, you can provide your own dispatcher name. It can be passed to appropriate variants of factory methods in
 `ReactiveKafka`: `publish()`, `producerActor()`, `producerActorProps()` or `consume()`, `consumerActor()`, `consumerActorProps()`.
 
 Testing
 ----
-Tests require Apache Kafka and Zookeeper to be available on localhost:9092 and localhost:2181
-
+Tests require Apache Kafka and Zookeeper to be available on `localhost:9092` and `localhost:2181`
+
+<!-- I'd start with the tests. We often want to just run tests as the first thing when looking at an open source
+project, to see if it's stable. This also tests the instructions - if the tests fail for us, and there's no indication
+on system requirements, the instructions are not that great. Will help you with this. AKa060216
+
+Suggested order for the README:
+
+- project scope / purpose (is already there)
+- Getting Started (new, includes requirements, how to run tests)
+- Using (includes lines to add in `build.sbt`)
+- Passing Configuration Properties to Kafka (as now)
+- Controlling consumer start offset (as now)
+- Working with actors (as now)
+- Tuning (as now; could be called Configuration?)
+-->
```

Comment:
The suggested order looks good. As for tests, currently to run them you must have Kafka + Zookeeper available at localhost with default ports which was "good enough" up to the point where we have more and more contributors.  Our options are dockerizing (like you mentioned) or finishing #38.